lib/chef/knife/status.rb

```diff
@@ -68,12 +72,21 @@ def run
         append_to_query("chef_environment:#{config[:environment]}") if config[:environment]
 
         if config[:hide_healthy]
+          ui.warn("-H / --hide-healthy is deprecated. Use --hide-by-mins MINS instead")
```

Comment:
I wonder if there's value in standardizing to using `Chef::Log.deprecation` in all these cases. It would present a unique opportunity to put a hook to do something when a deprecation is hit.